lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -38,6 +38,27 @@ class Map
   end
 end
 
+module Google
+  module Auth
+    # Extract project_id in initialize.
+    class ServiceAccountCredentials
+      singleton_class.send(:alias_method, :super_make_creds, :make_creds)
+      def self.make_creds(options = {})
+        json_key_io, scope = options.values_at(:json_key_io, :scope)
+        if json_key_io
+          json_key = MultiJson.load(json_key_io.read)
```

Comment:
Similar comment applied here, I'm just trying to understand why we're using the world "key" here, would it be fair to rename this to `json_credentials`?

Reply:
Same here, this is the JSON form of the private key (see the above code reference).